compiler/src/iree/compiler/Codegen/VMVX/LowerLinalgMicrokernels.cpp

```diff
@@ -99,9 +128,11 @@ struct StridedBufferDescriptor {
   Type getElementType() { return memRefType.getElementType(); }
   TypeAttr getElementTypeAttr() { return TypeAttr::get(getElementType()); }
 
-  /// Returns whether the innermost stride is statically 1. Many kernels
-  /// require this, so we provide the convenience here.
-  bool isUnitInnerStride() { return isMemRefUnitInnerStride(memRefType); }
+  // Returns true if all inner dimensions (that is, all but the outer-most dim)
+  // are statically known to be contiguous row-major.
+  bool areInnerDimsContiguousRowMajor() const {
+    return ::mlir::iree_compiler::areInnerDimsContiguousRowMajor(memRefType);
```

Comment:
What if we just rename the helper function above to avoid the name aliasing and remove the need for the full namespace qualification. verifyMemRefInnerDimsContiguousRowMajor?

Reply:
done